pkgs/applications/misc/darkman/default.nix

```
@@ -0,0 +1,49 @@
+{ lib, fetchFromGitLab, buildGoModule, scdoc, nix-update-script }:
+
+buildGoModule rec {
+  pname = "darkman";
+  version = "1.3.1";
+  nativeBuildInputs = [ scdoc ];
+
+  src = fetchFromGitLab {
+    owner = "WhyNotHugo";
+    repo = "darkman";
+    rev = "v${version}";
+    sha256 = "09iwc9cwwc88c6yrf6a552nbsnf1w8cnlra9axsar2p0k21v5yl1";
+  };
+
+  vendorSha256 = "09rjqw6v1jaf0mhmycw9mcay9q0y1fya2azj8216gdgkl48ics08";
+
+  postPatch = ''
+    substituteInPlace darkman.service \
+      --replace "/usr/bin/darkman" "$out/bin/darkman"
+    substituteInPlace contrib/dbus/nl.whynothugo.darkman.service \
+      --replace "/usr/bin/darkman" "$out/bin/darkman"
+    substituteInPlace contrib/dbus/org.freedesktop.impl.portal.desktop.darkman.service \
+      --replace "/usr/bin/darkman" "$out/bin/darkman"
+  '';
+
+  buildPhase = ''
+    runHook preBuild
+    make build
+    runHook postBuild
+  '';
+
+  installPhase = ''
+    runHook preInstall
+    make DESTDIR=$out PREFIX=/ install
```

We do not use DESTDIR often, indeed it is bad practice in Nixpkgs.

Usually `DESTDIR` is used as a "before targz-ing the package" in other distros. However, Nixpkgs installs things directly on Nix Store.

You should therefore do something like 

```suggestion
    make PREFIX=$out install
```

Or even better, set `makeFlags = [ "PREFIX=$(out)"  ];`

---

> Or even better, set makeFlags = [ "PREFIX=$(out)" ];

Please do this because it does not overwrite other things done in installPhase

---

Doesn't seem to be an option with `buildGoModule`? Correct me if I'm wrong.